src/carcockpit/gui.cpp

```diff
@@ -125,44 +125,83 @@ utki::shared_ref<ruis::key_proxy> carcockpit::make_root_widgets(utki::shared_ref
                             )
                         }
                     ),
-                    m::car_widget(c,
+                    m::gltf_viewer_widget(c,
                         {
                             .widget_params = {
-                                .id = "car_widget"s,
+                                .id = "gltf_viewer_widget_1"s,
                                 .lp = {
                                     .dims = {lp::fill, lp::fill},
                                     .weight = 5 // NOLINT(cppcoreguidelines-avoid-magic-numbers)
                                 },
-                                .clip=true
+                                .clip = true,
+                                .depth = true
+                            }
+                            ,
+                            .gltf_params = {
+                                .path_to_gltf = "../res/samples_gltf/spray.glb"s,
+                                // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers)
+                                .scaling_factor = 10.0f,
+                                // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers)
+                                .camera_target = ruis::vec3(1, 1, 1),
+                                .smooth_navigation_orbit = true,
+                                .smooth_navigation_zoom = true,
+                                .orbit_angle_upper_limit = M_PI_4,
```

Comment:
use `utki::pi / 4`, suppress lint if it complains about magic number

Reply:
done